lib/kitchen/docker/helpers/dockerfile_helper.rb

```diff
@@ -114,6 +118,26 @@ def rhel_platform
           CODE
         end
 
+        def centosstream_platform
```

Comment:
Is there any reason that we can't just treat centos stream like every other RHEL-compatible platform supported here? I see you're not installing curl, but I wouldn't expect the yum command from `rhel_platform` to fail here.

Reply:
@RulerOf Thanks for the review.  In my experience the yum command did fail because `curl` was already installed, in the centosstream 9 image this resulted in an error.

```
Error: 
 Problem: problem with installed package curl-minimal-7.76.1-14.el9.x86_64
  - package curl-minimal-7.76.1-14.el9.x86_64 conflicts with curl provided by curl-7.76.1-14.el9.x86_64
  - package curl-minimal-7.76.1-12.el9.x86_64 conflicts with curl provided by curl-7.76.1-14.el9.x86_64
  - cannot install the best candidate for the job
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
STDERR: The command '/bin/sh -c yum install -y sudo openssh-server openssh-clients which curl' returned a non-zero code: 1
```

Reply:
Didn't know about this package conflict. Thanks!